src/OrchardCore.Modules/OrchardCore.Admin/Startup.cs

```diff
@@ -81,4 +85,19 @@ public override void ConfigureServices(IServiceCollection services)
             });
         }
     }
+
+    [RequireFeatures("OrchardCore.Deployment")]
+    public class DeploymentStartup : StartupBase
+    {
+        public override void ConfigureServices(IServiceCollection services)
+        {
+            services.AddTransient<IDeploymentSource, SiteSettingsPropertyDeploymentSource<AdminSettings>>();
+            services.AddScoped<IDisplayDriver<DeploymentStep>>(sp =>
+            {
+                var S = sp.GetService<IStringLocalizer<Startup>>();
+                return new SiteSettingsPropertyDeploymentStepDriver<AdminSettings>(S["Admin settings"], S["Exports the admin site settings."]);
```

Comment:
The text here should probably be
`Exports the admin settings`

Users, shouldn't need to know these are stored on the `site` property. That's why we're doing the seperate steps this way, is so that users/admins don't need to understand the distinction.

It's ok that developers need to understand

Reply:
ok